python/tvm/relay/backend/contrib/ethosu/util.py

```diff
@@ -138,6 +139,12 @@ def round_up(a: int, b: int) -> int:
     return ((a + b - 1) // b) * b
 
 
+def get_accelerator_config():
+    """Get the variant of the accelerator to compile for"""
```

Comment:
Could use some more clarification here as well.

Reply:
this makes sense when you know there are multiple sizes of NPU but curious what the returned data type is and where i might go to read more about the things that could be present here.